cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -61,8 +61,6 @@ namespace cooperative_lanechange
         twist_sub_ = nh_->subscribe("current_velocity", 1, &CooperativeLaneChangePlugin::twist_cd, this);
         incoming_mobility_response_ = nh_->subscribe("incoming_mobility_response", 1 , &CooperativeLaneChangePlugin::mobilityresponse_cb, this);
         
-        georeference_sub_ = nh_->subscribe("georeference", 1, &CooperativeLaneChangePlugin::georeference_callback, this);
```

Comment:
The entire change set in this file is seems to be reverting the GPS fix which I completed last week. It seems to suggest an incorrect merge. If you don't know how to properly merge something you need to ask for assistance. Accidently reverting fixes is extremely problematic.